tools/packaging/common/envoy_bootstrap.json

```diff
@@ -192,12 +192,13 @@
   "static_resources": {
     "clusters": [
       {
-        "name": "prometheus_stats",
+        "name": "prometheus",
```

Comment:
I don't see any reason to change this? This seems to be risking breaking users for no benefit

Reply:
You are right.